galasa-ui/src/components/test-runs/TestRunsTable.tsx

```diff
@@ -120,6 +143,14 @@ export default function TestRunsTable({rawRuns}: {rawRuns: Run[]}) {
     return text;
   }, [rawRuns]);
 
+  if (isError) {
+    return <ErrorPage />;
+  }
+
+  if (isLoading) {
+    return <Loading small={false} active={isLoading}/>;
+  }
```

Comment:
I know this is likely similar to another page, but I think showing a table skeleton while the test runs load would be nicer than a circular loading icon. See https://carbondesignsystem.com/components/data-table/usage/#loading for an example - something similar to that would be good!